compiler/cpp/src/thrift/generate/t_swift_generator.cc

```diff
@@ -2249,47 +2260,23 @@ void t_swift_generator::render_const_value(ostream& out,
     }
 
     out << "]";
-
```

Comment:
[nit] do we need the changes here? I don't think I'm necessarily against them since we shouldn't get to this point for our telemetry thrift file, but just wondering

Reply:
We need the "set" else if branch to exist to support the set type constants, so I removed list conversion to keep it consistent with "set" throughout. The "render_const_value" is needed to be able to assign values statically in Thrift. 

I could leave the list functionality here, I would just have to also support list in the type declarations a bit earlier in the file